core-web/apps/dotcms-ui/src/app/view/components/dot-content-type-selector/dot-content-type-selector.component.html

```diff
@@ -0,0 +1,2 @@
+<p-dropdown [options]="options | async" [style]="{ width: '155px' }" (onChange)="change($event)">
```

Comment:
Let's add this:
<img width="400" alt="image" src="https://user-images.githubusercontent.com/751424/217033843-d541dd6a-898b-47a4-a3a4-54804999132f.png">

Because there could be hundreds of options.

Reply:
updated